relay-profiling/tests/fixtures/profiles/sample/no_transaction.json

```diff
@@ -0,0 +1,84 @@
+{
+  "event_id": "41fed0925670468bb0457f61a74688ec",
+  "version": "1",
+  "os": {
+    "name": "iOS",
+    "version": "16.0"
+  },
+  "device": {
+    "architecture": "arm64e",
+    "is_emulator": false,
+    "locale": "en_US",
+    "manufacturer": "Apple",
+    "model": "iPhone14,3"
+  },
+  "timestamp": "2022-09-01T09:45:00.000Z",
+  "profile": {
+    "samples": [
+      {
+        "stack_id": 0,
+        "thread_id": "1",
+        "queue_address": "0x0000000102adc700",
+        "relative_timestamp_ns": "10500500"
```

Comment:
@phacops since this is not really a timestamp, should we call it elapsed_ns instead to be more accurate?

Reply:
Let's call it `elapsed_since_start_ns`, makes it very clear that way. @viglia @armcknight